src/GeositeFramework/js/Plugin.js

```diff
@@ -719,6 +680,62 @@ require(['use!Geosite',
             return pluginObj.hasHelp;
         }
 
+        function printSetup(map, pluginObject, $printSandbox, modalConfirmDeferred, postModalDeferred) {
```

Comment:
its really hard to tell without comparing line by line -- is there any new code in `printSetup()` or `printReset()` ?

Reply:
Nope! I just separated them out into their own functions so they can be called independently of the modal.